flow/designs/nangate45/bp_be_top/config.mk

```diff
@@ -2,6 +2,17 @@ export DESIGN_NICKNAME = bp_be
 export DESIGN_NAME = bp_be_top
 export PLATFORM    = nangate45
 
+export SYNTH_HIERARCHICAL = 1
+export MAX_UNGROUP_SIZE = 100000
```

Comment:
any reason for two of the `MAX_UNGROUP_SIZE` be a  direct assignment `=` and the others `?=`? Also, should we consider setting some of these parameters per technology node or globally and overwriting them per design as needed to avoid repetition?

Reply:
Moved MAX_UNGROUP_SIZE to the platform/config.mk  and removed it from design config files.  Also the other settings will be removed before the merge